src/ng-xform/fields/field-group.ts

```diff
@@ -0,0 +1,17 @@
+import { DynamicField } from './dynamic-field';
+
+export class FieldGroup extends DynamicField {

```

Comment:
`FieldGroup` is a way to edit a nested object. `NestedObjectField` would be more correct